core/src/main/java/io/seata/core/rpc/netty/processor/client/RmHandleBranchRollbackProcessor.java

```diff
@@ -0,0 +1,73 @@
+/*
+ *  Copyright 1999-2019 Seata.io Group.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package io.seata.core.rpc.netty.processor.client;
+
+import io.netty.channel.ChannelHandlerContext;
+import io.seata.common.util.NetUtil;
+import io.seata.core.protocol.RpcMessage;
+import io.seata.core.protocol.transaction.BranchRollbackRequest;
+import io.seata.core.protocol.transaction.BranchRollbackResponse;
+import io.seata.core.rpc.RemotingClient;
+import io.seata.core.rpc.TransactionMessageHandler;
+import io.seata.core.rpc.netty.processor.NettyProcessor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The rm client handle branch commit processor
+ * <p>
+ * handle TC do global rollback command.
+ * {@link BranchRollbackRequest}
+ *
+ * @author zhangchenghui.dev@gmail.com
+ * @since 1.2.0
+ */
+public class RmHandleBranchRollbackProcessor implements NettyProcessor {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(RmHandleBranchRollbackProcessor.class);
+
+    private TransactionMessageHandler handler;
+
+    private RemotingClient remotingClient;
+
+    public RmHandleBranchRollbackProcessor(TransactionMessageHandler handler, RemotingClient remotingClient) {
+        this.handler = handler;
+        this.remotingClient = remotingClient;
+    }
+
+    @Override
+    public void process(ChannelHandlerContext ctx, RpcMessage rpcMessage) throws Exception {
+        String remoteAddress = NetUtil.toStringAddress(ctx.channel().remoteAddress());
+        Object msg = rpcMessage.getBody();
+        if (LOGGER.isInfoEnabled()) {
+            LOGGER.info("rm handle branch rollback process:" + msg);
+        }
+        handleBranchRollback(rpcMessage, remoteAddress, (BranchRollbackRequest) msg);
+    }
+
+    private void handleBranchRollback(RpcMessage request, String serverAddress, BranchRollbackRequest branchRollbackRequest) {
+        BranchRollbackResponse resultMessage;
+        resultMessage = (BranchRollbackResponse) handler.onRequest(branchRollbackRequest, null);
+        if (LOGGER.isDebugEnabled()) {
+            LOGGER.debug("branch rollback result:" + resultMessage);
+        }
+        try {
+            this.remotingClient.sendResponse(request, serverAddress, resultMessage);
+        } catch (Throwable throwable) {
+            LOGGER.error("send response error: {}", throwable.getMessage(), throwable);
```

Comment:
Why is the exception handling different from RmHandleBranchCommitProcessor?

Reply:
This is the old code. 
There are BranchCommitResponse and BranchRollbackResponse for customer response to the server request, but the server does anything to the response request, so I think the exception handling of RmBranchCommitProcessor is meaningless, so I will change it to the same as RmBranchRollbackProcessor .

Reply:
Done.